frontend_build/src/read_bundle_plugins.js

```diff
@@ -24,8 +24,6 @@ var libs = function(kolibri_name) {
     'logging': kolibri_name + '.lib.logging',
     'vue': kolibri_name + '.lib.vue',
     'kolibri': kolibri_name,
-    'core-base': kolibri_name + '.lib.coreBase',
```

Comment:
Is there a chance that a lot of code that runs in my dev environment is broken in production because I haven't been adding core libs to the `Lib` object?